scripts/data_guide/data_guide.qmd

```diff
@@ -269,7 +269,7 @@ Samples that have variants that could be merged together will be filtered out. T
 
 [Genome Nexus Annotation Status]{.underline}
 
-Variants that are unable to be annotated by Genome Nexus are removed.
+Mutation data that fail annotation by Genome-Nexus are excluded from the release. For example, this occurs when the allele extracted from the VCF or MAF file does not match the reference allele, or when the VEP tool is unable to annotate the data.
```

Comment:
```suggestion
Mutation data that fails to be annotation by Genome-Nexus is excluded from the release. For example, this occurs when the allele extracted from the VCF or MAF file does not match the reference allele, or when the VEP tool is unable to annotate the data.
```

Reply:
A combination probably works: `Mutation data that fails to be annotated by Genome-Nexus is excluded from the release.`